hil-test/src/bin/esp_radio_floats.rs

```diff
@@ -123,19 +123,6 @@ mod tests {
             .start_app_core(
                 unsafe { &mut *core::ptr::addr_of_mut!(APP_CORE_STACK) },
                 move || {
-                    // app core starts with interrupts disabled
```

Comment:
The HAL is supposed to set up and enable the interrupts it needs, none of this seem to make a difference.